test/integration/collection-management/promise_collection_db_management.test.ts

```diff
@@ -1,6 +1,6 @@
 import { assert as test, setupDatabase } from '../shared';
```

Comment:
let's rename this file to get rid of `promise` 😄

Reply:
done